src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/ExtensionDataObject.cs

```diff
@@ -13,15 +13,15 @@ public sealed class ExtensionDataObject
     {
         private IList<ExtensionDataMember> _members;
 
-#if USE_REFEMIT
+#if USE_REFEMIT || NET_NATIVE
```

Comment:
Do we even need the USE_REFEMIT #ifdefs? It doesn't look like we are defining them on the csproj at all so I wonder if we should just clean those out and keep only the NET_NATIVE ones.